jersey/src/main/java/com/netflix/conductor/server/resources/WebAppExceptionMapper.java

```diff
@@ -32,7 +33,9 @@
 
 	@Override
 	public Response toResponse(WebApplicationException exception) {
-		return exception.getResponse();
+
+        return Response.status(Response.Status.BAD_REQUEST).entity(exception.getResponse()).build();
```

Comment:
Not correct to map all WebApplicationException to BAD_REQUEST, map it to ErrorResponse appropriately so we return the same JSON error response back in this case.